.pre-commit-config.yaml

```diff
@@ -1,5 +1,4 @@
 ci:
-  autofix_prs: false
```

Comment:
> Personally, the pre-commit.ci autofix does save the exact situation that we're initially in, where we banked a PR that wasn't compliant for all the git-hooks

With this flag as false, pre-commit would still show a failure in the PR CI, and then you can choose whether to have it fix automatically by summoning the bot ([example](https://github.com/matplotlib/matplotlib/pull/25081#issuecomment-1455195377)).

Reply:
Seems like a happy compromise :+1: